test/CodeGenILTests/ReplicationTests.cs

```diff
@@ -488,7 +492,7 @@ public void MSIL_Replication_BuiltinMethods()
             var output = codeGen.EmitAndExecute(ast);
             Assert.IsNotEmpty(output);
 
-            Assert.AreEqual(output["test1"], new Object[] { true, true });
+            Assert.AreEqual(output["test1"], false);
```

Comment:
you verified this is the correct result?

Reply:
yes